lib/rules/indent.js

```diff
@@ -168,7 +168,7 @@ module.exports = {
                 let rangeToFix = [];
 
                 if (needed > gotten) {
-                    const spaces = "" + new Array(needed - gotten + 1).join(indentChar);  // replace with repeat in future
+                    const spaces = `${new Array(needed - gotten + 1).join(indentChar)}`;  // replace with repeat in future
```

Comment:
You're right-- and to be honest, I was questioning the leading `""` (from the old code) as well for that same reason. But I just thought there was an important reason for it and didn't think about it too closely. :confused: